jdk/src/share/classes/sun/nio/cs/ext/ExtendedCharsets.java

```diff
@@ -1202,6 +1205,16 @@ protected void init() {
                 }
             }
         }
+        if (isGB18030_2000) {
+            deleteCharset("GB18030",
+                new String[] {
+                    "gb18030-2022"
+                });
+            charset("GB18030", "GB18030",
+                new String[] {
+                    "gb18030-2000"
+                });
+        }
```

Comment:
Can you please add this replacement to the comment above listing the changes done in init?

Reply:
Good catch. I've updated the comment to document `jdk.charset.GB18030`.